e2e-polybft/helpers_test.go

```diff
@@ -77,3 +80,48 @@ func getRootchainValidators(relayer txrelayer.TxRelayer, checkpointManagerAddr,
 
 	return validators, nil
 }
+
+func hasValidatorSealed(srv *framework.TestServer, firstblock, lastblock uint64, validators polybft.AccountSet, validatorAddr ethgo.Address) (bool, error) {
+	for i := firstblock + 1; i <= lastblock; i++ {
+		block, err := srv.JSONRPC().Eth().GetBlockByNumber(ethgo.BlockNumber(i), false)
+		if err != nil {
+			return false, err
+		}
+
+		extra, err := polybft.GetIbftExtra(block.ExtraData)
+		if err != nil {
+			return false, err
+		}
+
+		validatorssignatures, _ := validators.GetFilteredValidators(extra.Parent.Bitmap)
+		if validatorssignatures.ContainsAddress(types.Address(validatorAddr)) {
+			return true, nil
+		}
+	}
+
+	return false, nil
+}
+
+func waitForNonZeroBalance(srv *framework.TestServer, address ethgo.Address, dur time.Duration) (*big.Int, error) {
```

Comment:
Same here, member of `TestServer`?

Reply:
fixed: https://github.com/0xPolygon/polygon-edge/pull/1258/commits/280ec4c1235e74c2e4e08d3b527915863bfc3d85